### DISCUSSION TITLE
Unable to create new project

### DISCUSSION THREAD

**Discussion by joweste** [nuxt/nuxt#9902] [Questions (Nuxt 2)]
Hi, I am trying create a nuxt app, but I get the next errors.
In  the past, I am used to create nuxt app without problems.
I am  tried install without eslint and prettier and I get problems as well as posted below.

My npm version is:
F:\Programas\vuejs\nuxt-iclindoctor>npm --version
7.24.1
and node version is:
F:\Programas\vuejs\nuxt-iclindoctor>node --version
v16.9.1

What am I doing wrong?

F:\Programas\vuejs\nuxt-iclindoctor>npx create-nuxt-app  site-iclindoctor

create-nuxt-app v3.7.1
✨  Generating Nuxt.js project in site-iclindoctor
? Project name: site-iclindoctor
? Programming language: JavaScript
? Package manager: Npm
? UI framework: Vuetify.js
? Nuxt.js modules: Axios - Promise based HTTP client, Progressive Web App (PWA)
? Linting tools: ESLint, Prettier
? Testing framework: None
? Rendering mode: Universal (SSR / SSG)
? Deployment target: Static (Static/Jamstack hosting)
? Development tools: jsconfig.json (Recommended for VS Code if you're not using typescript)
? Continuous integration: None
? Version control system: None
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@nuxtjs%2feslint-config - Not found
npm ERR! 404
npm ERR! 404  '@nuxtjs/eslint-config@^6.0.1' is not in this registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Luiz-HP-PC\AppData\Local\npm-cache\_logs\2021-10-07T16_50_22_140Z-debug.log

C:\Users\Luiz-HP-PC\AppData\Local\npm-cache\_npx\ea7c6e7510ea3309\node_modules\sao\lib\installPackages.js:108
        throw new SAOError(`Failed to install ${packageName} in ${cwd}`)
        ^

SAOError: Failed to install packages in F:\Programas\vuejs\nuxt-iclindoctor\site-iclindoctor
    at ChildProcess.<anonymous> (C:\Users\Luiz-HP-PC\AppData\Local\npm-cache\_npx\ea7c6e7510ea3309\node_modules\sao\lib\installPackages.js:108:15)
    at ChildProcess.emit (node:events:394:28)
    at ChildProcess.cp.emit (C:\Users\Luiz-HP-PC\AppData\Local\npm-cache\_npx\ea7c6e7510ea3309\node_modules\sao\node_modules\cross-spawn\lib\enoent.js:34:29)
    at maybeClose (node:internal/child_process:1064:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:301:5) {
  __sao: true
}

// Withot eslint and prettier

F:\Programas\vuejs\nuxt-iclindoctor>npx create-nuxt-app  site-iclindoctor

create-nuxt-app v3.7.1
✨  Generating Nuxt.js project in site-iclindoctor
? Project name: site-iclindoctor
? Programming language: JavaScript
? Package manager: Npm
? UI framework: Vuetify.js
? Nuxt.js modules: Axios - Promise based HTTP client, Progressive Web App (PWA)
? Linting tools: (Press <space> to select, <a> to toggle all, <i> to invert selection)
? Testing framework: None
? Rendering mode: Universal (SSR / SSG)
? Deployment target: Static (Static/Jamstack hosting)
? Development tools: jsconfig.json (Recommended for VS Code if you're not using typescript)
? What is your GitHub username? luiz-pc-hp
? Version control system: None
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@npmcli%2ffs - Not found
npm ERR! 404
npm ERR! 404  '@npmcli/fs@^1.0.0' is not in this registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Luiz-HP-PC\AppData\Local\npm-cache\_logs\2021-10-07T17_03_47_028Z-debug.log

C:\Users\Luiz-HP-PC\AppData\Local\npm-cache\_npx\ea7c6e7510ea3309\node_modules\sao\lib\installPackages.js:108
        throw new SAOError(`Failed to install ${packageName} in ${cwd}`)
        ^

SAOError: Failed to install packages in F:\Programas\vuejs\nuxt-iclindoctor\site-iclindoctor
    at ChildProcess.<anonymous> (C:\Users\Luiz-HP-PC\AppData\Local\npm-cache\_npx\ea7c6e7510ea3309\node_modules\sao\lib\installPackages.js:108:15)
    at ChildProcess.emit (node:events:394:28)
    at ChildProcess.cp.emit (C:\Users\Luiz-HP-PC\AppData\Local\npm-cache\_npx\ea7c6e7510ea3309\node_modules\sao\node_modules\cross-spawn\lib\enoent.js:34:29)
    at maybeClose (node:internal/child_process:1064:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:301:5) {
  __sao: true
}

F:\Programas\vuejs\nuxt-iclindoctor>npm --version
7.24.1

F:\Programas\vuejs\nuxt-iclindoctor>node --version
v16.9.1

F:\Programas\vuejs\nuxt-iclindoctor>

**Comment by vicmars5** [comment]
I am having the same issue, also I found most of the `nuxtjs/` repos are no longer in npmjs.com.
https://www.npmjs.com/package/@nuxtjs/eslint-config

<img width="1440" alt="Screen Shot 2021-10-07 at 11 06 25" src="https://user-images.githubusercontent.com/5891887/136431439-05f01e36-0df0-4de2-8201-48aa5364d3c7.png">

**Comment by vicmars5** [comment]
For what I see the repos are back online again, I was able to install `@nuxtjs/eslint-config`

**Reply by joweste** [reply]
It seems come back to normal.
I just tested and works now
